.ci_support/linux_64_.yaml

```diff
@@ -1,10 +1,8 @@
-cdt_name:
-- conda
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.9'
+- '3.10'
```

Comment:
```suggestion
- '3.11'
```

Reply:
I bumped the python version because of this line in the [Azure build log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1343533&view=logs&jobId=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736):
`pip._internal.exceptions.UnsupportedPythonVersion: Package 'plopp' requires a different Python: 3.10.18 not in '>=3.11'`

Reply:
>Here: https://github.com/conda-forge/plopp-feedstock/commit/08041f105c3eb9e1f3c204d52756996a697ad507

That should be fine I think, it will just get overwritten the next time we do a conda smithy rendered (happens at every release). No need to fix anything :) 

The only bumps we need to do and manage are in `recipe/recipe.yaml` files for all conda releases. Everything else in these repos are just for conda-forge infrastructure.

Reply:
Why did the azure build fail then?